subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/samples/SamplesWebQuickstartIntegrationTest.groovy

```diff
@@ -15,16 +15,13 @@
  */
 
 package org.gradle.integtests.samples
+
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.integtests.fixtures.Sample
 import org.gradle.test.fixtures.file.LeaksFileHandles
 import org.gradle.test.fixtures.file.TestFile
 import org.gradle.util.ports.ReleasingPortAllocator
 import org.junit.Rule
-import spock.lang.Timeout
-import spock.lang.Unroll
-
-import static org.gradle.integtests.fixtures.UrlValidator.available
 
 @LeaksFileHandles
```

Comment:
We likely won't need this annotation anymore. It was only needed for the Jetty test case.